components/nif/nifstream.hpp

```diff
@@ -26,10 +26,44 @@ class NIFStream {
     /// Input stream
     Files::IStreamPtr inp;
 
-    uint8_t read_byte();
-    uint16_t read_le16();
-    uint32_t read_le32();
-    float read_le32f();
+    uint8_t read_byte() {
+        uint8_t byte;
+        inp->read((char*)&byte, 1);
+        return byte;
+    }
+
+    uint16_t read_le16() {
+        alignas(2) uint8_t buffer[2];
+        inp->read((char*)buffer, 2);
```

Comment:
Seems MSVC2013 doesn't handle ```alignas```. It's not necessary here anyway, you can do:
```
uint16_t ret;
inp->read(reinterpret_cast<char*>(&ret), 2);
return ret;
```
The same for le32, le64, and le32f. Also, these should be marked as inline if you want them inlined.